.github/workflows/check-pl-compat.yaml

```diff
@@ -116,21 +115,6 @@ jobs:
         RT_BUILD_DIR="$(pwd)/runtime-build" \
         QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
         QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
-        LIGHTNING_GIT_TAG_VALUE=master \
-        ENABLE_LIGHTNING_KOKKOS=ON \
-        ENABLE_OPENQASM=ON \
-        make runtime
-
-    - name: Build Lightning Runtime (stable)
-      if: ${{ inputs.lightning == 'stable' }}
-      run: |
-        COMPILER_LAUNCHER="" \
-        C_COMPILER=$(which gcc }}) \
-        CXX_COMPILER=$(which g++ }}) \
-        RT_BUILD_DIR="$(pwd)/runtime-build" \
-        QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
-        QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
-        LIGHTNING_GIT_TAG_VALUE=latest_release \
```

Comment:
Revert this to still build against stable Lightning

Reply:
I'm closing this since it's outdated now :)